cli/azd/internal/scaffold/spec_service_binding.go

```diff
@@ -271,6 +271,14 @@ func GetServiceBindingEnvsForPostgres(postgres DatabasePostgres) ([]Env, error)
 				Name:  "spring.datasource.azure.passwordless-enabled",
 				Value: "true",
 			},
+			{
```

Comment:
Just double confirm that: this is a complete Env list, then later it will be determined if added to bicep by us, or added by service connector, right?

In other words, before this PR, it doesn't block the MySql / Postgresql + MI auth, right?

Reply:
>  this is a complete Env list

yes.

> then later it will be determined if added to bicep by us, or added by service connector, right?

The items added by service connector will not added into bicep file, those items will be filtered out. you can check the diff in current PR. 

> ![image](https://github.com/user-attachments/assets/c67d1e4d-6392-44db-bba3-5328d080cbe3)

> In other words, before this PR, it doesn't block the MySql / Postgresql + MI auth, right?

Right.